### PR TITLE
man1/openssl.pod: add missing cipher aliases

### PR DESCRIPTION
**Pull request by evqna**
Following-up on issue #7273.

These appear to be some of the most widely used block/stream ciphers which supports adding them to the main man page (not sure about SM4 though).

Also adding notes to increase cross-linking inside the documentation.

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man1/openssl.pod [hunk 1]
```diff
@@ -40,6 +40,9 @@ The B<openssl> program provides a rich variety of commands (I<command> in the
 SYNOPSIS above), each of which often has a wealth of options and arguments
 (I<command_opts> and I<command_args> in the SYNOPSIS).
 
+Detailed documentation and use cases for most standard subcommands is available
+(eg. L<x509(1)> or L<openssl-x509(1)>).
```
Comment (InfoHunter) [inline doc/man1/openssl.pod]: It seems `eg.` should be `e.g.`

Comment (InfoHunter) [reply doc/man1/openssl.pod]: I guess it should be "...are available..."

Comment (kaduk) [reply doc/man1/openssl.pod]: "e.g.,", even (with comma)

Comment (InfoHunter) [reply doc/man1/openssl.pod]: Right, absolutely

**Comment by InfoHunter** [comment]
Thanks @paulidale for the second review and I will merge this since it has adequate reviews.

**Comment by InfoHunter** [comment]
Commits are squashed and pushed to master (1362190b1b) and 1.1.1(a2516f0c3e), thanks!

**Pull request closed without merging**